### PR TITLE
Make Invariant Fuzzer runnable against release v0.3.0

### PR DESCRIPTION
**Pull request by relyt29**
## Purpose of Changes and their Description

Makes updates to the invariant / fuzzer test suite so that inferences can actually be sent during a fuzz run

## Link(s) to Ticket(s) or Issue(s) resolved by this PR

No tickets created for this one

## Are these changes tested and documented?

These changes are tests themselves. They have no implications relevant to documentation

### CODE REVIEW

**Review by xmariachi — COMMENTED** [review]
Remove references to "leader" or "bulk"

**Review by guilherme-brandao — APPROVED** [review]

File: test/invariant/inferences_test.go [hunk 1]
```diff
@@ -15,8 +15,8 @@ import (
 
 func doInferenceAndReputation(
 	m *testcommon.TestConfig,
-	leaderWorker Actor,
-	leaderReputer Actor,
+	_ Actor,
```
Comment (xmariachi) [inline test/invariant/inferences_test.go:18]: why are these params needed to be passed in at all if they are not used? Can't we remove these params?

**Review by xmariachi — APPROVED** [review]
lgtm, left one comment

File: test/invariant/inferences_test.go (hunk 1 above)
Comment (relyt29) [reply test/invariant/inferences_test.go:18]: no they're part of an interface that is expected by the "list of actions a fuzz round can take"

some functions don't use them, some do, the ones that don't just ignore em

**Pull request merged**